Gordon360/Services/HousingService.cs

```diff
@@ -90,6 +90,31 @@ public bool RemoveHousingAdmin(string id)
             return true;
         }
 
+        /// <summary>
+        /// Calls a stored procedure that deletes the application with given id
+        /// (and consequently all rows that reference it)
+        /// </summary>
+        /// <param name="applicationID"> The id of the application to delete </param>
+        /// <returns> Whether or not this was successful </returns>
+        public bool DeleteApplication(int applicationID)
+        {
+            IEnumerable<ApartmentApplicationViewModel> result = null;
+
+            SqlParameter appIdParam = new SqlParameter("@APP_ID", applicationID);
+
+            result = RawSqlQuery<ApartmentApplicationViewModel>.query("DELETE_AA_APPLICATION @APP_ID", appIdParam);
```

Comment:
For DELETE stored procedures, use `_context.Database.ExecuteSqlCommand` (the `_context` will come in when you merge apartapp-develop into this branch)

Reply:
```suggestion
            int? result = _context.Database.ExecuteSqlCommand("DELETE_AA_APPLICATION @APP_ID", appIdParam);
```